spec/periodicChecksSpec.js

```diff
@@ -825,4 +913,206 @@ describe('Periodic Checks Module', () => {
+                  },
+                  message: 'changes',
+                  tree: {
+                    sha: 'f5f8be9b0e4ac9970f68d8945de3474581b20d03',
+                    url:
+                      'https://api.github.com/repos/jameesjohn/oppia/git/trees/f5f8be9b0e4ac9970f68d8945de3474581b20d03',
+                  },
+                  url:
+                    'https://api.github.com/repos/jameesjohn/oppia/git/commits/eb697e55a3da3085360d843dfb150f033aa17e61',
+                  comment_count: 0,
+                  verification: {},
+                },
+                url:
+                  'https://api.github.com/repos/oppia/oppia/commits/eb697e55a3da3085360d843dfb150f033aa17e61',
+                html_url:
+                  'https://github.com/oppia/oppia/commit/eb697e55a3da3085360d843dfb150f033aa17e61',
+                comments_url:
+                  'https://api.github.com/repos/oppia/oppia/commits/eb697e55a3da3085360d843dfb150f033aa17e61/comments',
+                author: {},
+                committer: {},
+                parents: [],
+              },
+            };
+          }
+          return {
+            data: {
+              sha: 'new-build-pr-sha',
+              node_id:
+                'MDY6Q29tbWl0MTczMDA0MDIyOjUyNWQ2MDU4YTYyNmI0NjE1NGVkMzczMTE0MWE5NWU3MGViYjBhZWY=',
+              commit: {
+                author: {
+                  name: 'James James',
+                  email: 'jamesjay4199@gmail.com',
+                  date: '2020-08-13T13:43:24Z',
+                },
+                committer: {
+                  name: 'James James',
+                  email: 'jamesjay4199@gmail.com',
+                  date: '2020-08-13T13:43:24Z',
+                },
+                message: 'new additions',
+                tree: {
+                  sha: 'b5bf5af6ec0592bf3776b23d4355ff200549f427',
+                  url:
+                    'https://api.github.com/repos/jameesjohn/oppia/git/trees/b5bf5af6ec0592bf3776b23d4355ff200549f427',
+                },
+                url:
+                  'https://api.github.com/repos/jameesjohn/oppia/git/commits/525d6058a626b46154ed3731141a95e70ebb0aef',
+                comment_count: 0,
+                verification: {
+                  verified: false,
+                  reason: 'unsigned',
+                  signature: null,
+                  payload: null,
+                },
+              },
+              url:
+                'https://api.github.com/repos/jameesjohn/oppia/commits/525d6058a626b46154ed3731141a95e70ebb0aef',
+              html_url:
+                'https://github.com/jameesjohn/oppia/commit/525d6058a626b46154ed3731141a95e70ebb0aef',
+              comments_url:
+                'https://api.github.com/repos/jameesjohn/oppia/commits/525d6058a626b46154ed3731141a95e70ebb0aef/comments',
+              author: [Object],
+              committer: [Object],
+              parents: [Array],
+            },
+          };
```

Comment:
Could you extract the commit data into separate variables?

Reply:
Done.